common/src/main/java/com/wynntils/models/containers/LootChestModel.java

```diff
@@ -112,19 +112,25 @@ public void onScreenInit(ScreenInitEvent.Pre e) {
 
     @SubscribeEvent
     public void onSetSlot(ContainerSetSlotEvent.Post event) {
-        if (event.getContainerId() != nextExpectedLootContainerId) return;
-        if (event.getSlot() >= LOOT_CHEST_ITEM_COUNT) return;
+        if (event.getContainerId() != nextExpectedLootContainerId) {
+            return;
+        }
+        if (event.getSlot() >= LOOT_CHEST_ITEM_COUNT) {
+            return;
+        }
 
         ItemStack itemStack = event.getItemStack();
 
         processItemFind(itemStack);
 
         Optional<GearBoxItem> gearBoxItem = Models.Item.asWynnItem(itemStack, GearBoxItem.class);
-        if (gearBoxItem.isEmpty()) return;
+        if (gearBoxItem.isEmpty()) {
```

Comment:
Please don't make changes to existing files that is not needed for the patch to work. Especially not syntax related ones. Your changes does in fact break a long-standing tradition of trying to "short-circuit" out by using non-braced returns on the same line as the if statement.

Reply:
Yes, sorry, that's my vscode autoformatter being pesky. I've ended up turning it off to avoid these needless changes.